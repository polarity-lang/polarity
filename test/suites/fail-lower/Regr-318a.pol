data Bool { True, False }
data T { K }

let foo: T { K(True)}