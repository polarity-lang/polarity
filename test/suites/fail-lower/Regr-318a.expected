L-014

  × Too many arguments provided
   ╭─[Regr-318a.pol:3:1]
 3 │ 
 4 │ let foo: T { K(True)}
   ·                ────
   ╰────
